docs/book/collaborate/collaborate.md

```diff
@@ -0,0 +1,65 @@
+---
+description: Collaboration with ZenML.
+---
+
+# ZenML for Teams and Organizations
+
+Using ZenML to develop and execute pipelines from the comfortable confines of
+your workstation or laptop is a great way to incorporate MLOps best practices
+and production grade quality into your project from day one. However, as Machine
+Learning projects grow, managing and running ZenML in a single-user and
+single-machine setting can become a strenuous task. More demanding projects
+involve creating and managing an increasing number of Pipelines and complex
+Stacks built on a wider and continuously evolving set of technologies. This can
+easily exceed the capabilities of a single machine and person or role.
+
+The same principles observed in the ZenML single-user experience are applicable
+as a framework of collaboration between the various specialized roles in the
+AI/ML team. We purposefully maintain a clean separation between the core ZenML
+concepts, with the intention that they may be managed as individual
+responsibilities assigned to different members of a team or department without
+incurring the overhead and friction usually associated with larger
+organizations.
+
+One such example is the decoupling of ZenML Stacks from Pipelines and their
+associated code, a principle that provides for a smooth transition from
+experimenting with and running Pipelines locally to deploying them to production
+environments. It is this same decoupling that also allows ZenML to be used
+in a setting where part of a team is iterating on writing ML code and
+implementing Pipelines, while the other part is defining and actively
+maintaining the infrastructure and the Stacks that will be used to execute
+Pipelines in production. Everyone can remain focused on their responsibilities
```

Comment:
Same here (as elsewhere in this doc)... pipelines are common enough a term. Do we need to have it capitalised?

Reply:
agreed

Reply:
My feeling was that Stefan was trying to denote ZenML-specific concepts with capitals. On the face of it, it's not a terrible idea. The only criterion in my mind is whether reading it as docs makes sense or not.